### PR TITLE
[Components v3] Form label

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

New component : form label

-----

image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2202/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5212/ui-diff)

File: packages/scss/src/components/formLabel/component.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@mixin component($atRoot: 'without: rule') {
+  color: var(--components-formLabel-color);
+  display: inline-block;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/formLabel/component.scss:3]: Maybe `display: inline-flex` + `gap` + `align-items` here ? For consistency with others similars components

Comment (jeremie-lucca) [reply packages/scss/src/components/formLabel/component.scss:3]: That's the first way I tried, but even by playing with wrap I did not success to have a good line breaking management, including asterisk and "?" icon.

**Review by BertrandPodevin — APPROVED** [review]

**Review by GuillaumeNury — DISMISSED** [review]
Juste une mini-modif si tu veux ne pas avoir les espaces inutiles sur ZH

**Review by GuillaumeNury — APPROVED** [review]

**Review by BertrandPodevin — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2202/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/5242/ui-diff)

**Pull request merged**